README.md

```diff
@@ -80,6 +80,41 @@ export class MoviesStore extends ImmerComponentStore<MoviesState> {
 }
 ```
 
+## `immerPatchState`
+
+Provides an Immer-version of the Signal Store's `patchState`. It adds an additional updater function
+which can mutate the state.
+
+```ts
+const UserState = signalStore(withState({
+	id: 1,
+	name: { firstname: 'Konrad', lastname: 'Schultz' },
+	address: { city: 'Vienna', zip: '1010' },
+}), withComputed(({ name }) => ({ prettyName: computed(() => `${name.firstname()} ${name.lastname()}`) })));
+
+const userState = new UserState();
+
+immerPatchState(userState, (state => {
+	state.name = { firstname: 'Lucy', lastname: 'Sanders' };
+	state.address.zip = '1020'
+}));
```

Comment:
In addition to formatting, I suggest providing an example that is close to real-world scenarios (without manual store instantiation), because beginners may think that this is the right/recommended way of doing things. We can also remove unnecessary parts from this example such as `withComputed`. Also, note that `UserState` is renamed to `UserStore`.

```suggestion
const UserStore = signalStore(
  withState({
    user: { firstName: 'Konrad', lastName: 'Schultz' },
    address: { city: 'Vienna', zip: '1010' },
  }),
  withMethods((store) => ({
    setLastName(lastName: string): void {
      immerPatchState(store, (state) => {
        state.user.lastName = lastName;
      });
    },
    setCity(city: string): void {
      immerPatchState(store, (state) => {
        state.address.city = city;
      });
    },
  }))
);
```

Another option is to use `signalState` instead of `signalStore` in the example:

```suggestion
const userState = signalState({
    user: { firstName: 'Konrad', lastName: 'Schultz' },
    address: { city: 'Vienna', zip: '1010' },
});

immerPatchState(userState, (state) => {
 // ...
});
```